dependency_versions.py

```diff
@@ -10,7 +10,7 @@
 
 # This is the "version" of the whole base_deps_versions dict.
 # Change this when you change base_deps_versions.
-base_deps_meta_version = '1'
+base_deps_meta_version = '1.0.1'
```

Comment:
Please, keep only on number to this version number. This is just an "internal" incrementing number to be sure that travisCI get the right up-to-date archive with the prebuild dependencies.

If you increment this number, it is because the `base_deps_versions` has changed and it is not the case.
Please also update the `Gradle` entry in `base_deps_versions` (yes, the version of number of Gradle was not matching the real version used, but it is the occasion to fix this :stuck_out_tongue:  )